hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -547,7 +553,7 @@ public SnapshotDiffResponse getSnapshotDiffReport(
           new SnapshotDiffReportOzone(snapshotRoot.toString(), volumeName,
               bucketName, fromSnapshotName, toSnapshotName, new ArrayList<>(),
               null),
-          CANCELLED, 0L, JobCancelResult.CANCELLATION_SUCCESS);
+          CANCELLED, 0L, null);
```

Comment:
this switch case is for "job status == CANCELLED", should the 'null' be 'CANCEL_ALREADY_CANCELLED_JOB'?

Reply:
I think it is be nothing similar to In_progress, Done and others. I'll update it.

Thanks for the catch.

Reply:
sure!